project_close_forecast_date/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Close Forecast Date",
+    "summary": """
+        Adds new field forecast date in purchase order line.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Project",
+    "website": "https://github.com/solvosci/slv-project",
+    "depends": ["project_purchase"],

```

Comment:
This dependency is actually unused (`project_purchase` only adds a soft link between purchases and projects, nothing that seems to be used here), and stock dependency (due to `stock_picking`) is missing, then better try this:
```suggestion
    "depends": ["project", "purchase_stock"],

```